src/index.tsx

```diff
@@ -358,6 +360,43 @@ function deleteSelectedElements() {
   }
 }
 
+function save() {
+  if (elements && elements.length > 0) {
+    const items = [...elements];
+    for (const item of items) {
+      item.isSelected = false;
```

Comment:
I'm curious, why are you removing the selection? It may not be a bad idea to restore it as well.